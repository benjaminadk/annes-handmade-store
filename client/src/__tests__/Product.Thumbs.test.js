import React from 'react'
import Thumbs from '../components/Product.Thumbs'
import { shallow } from 'enzyme'

const images = ['https://shopping-site.s3.amazonaws.com/images/og-image']

const activeIndex = 0

test('Product Thumbs Component renders correctly', () => {
  const wrapper = shallow(<Thumbs images={images} activeIndex={activeIndex} />)
  expect(wrapper.props().images).toEqual(images)
  expect(wrapper).toMatchSnapshot()
})
