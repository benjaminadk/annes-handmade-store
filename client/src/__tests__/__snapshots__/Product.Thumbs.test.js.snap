// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Thumbs Component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(Thumbs)
    activeIndex={0}
    images={
        Array [
            "https://shopping-site.s3.amazonaws.com/images/og-image",
          ]
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "activeIndex": 0,
      "classes": Object {
        "root": "Thumbs-root-1",
        "thumb": "Thumbs-thumb-2",
      },
      "images": Array [
        "https://shopping-site.s3.amazonaws.com/images/og-image",
      ],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeIndex": 0,
        "classes": Object {
          "root": "Thumbs-root-1",
          "thumb": "Thumbs-thumb-2",
        },
        "images": Array [
          "https://shopping-site.s3.amazonaws.com/images/og-image",
        ],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
